server/src/controllers/auth.js

```diff
@@ -0,0 +1,36 @@
+const User = require('../models/user');
+const jwt = require("jsonwebtoken");
+
+exports.signup = ( req, res) => {
```

Comment:
This is not how you are supposed to export in this project. Kindly refer to the `example.controller.js` file which you have deleted and should not be deleted as it is for reference purposes at the moment which can be removed after 2-3 endpoints are constructed properly with conventions.

Same goes for `example.model.js` and `example.route.js`

And rename `auth.js` to `auth.controller.js`

Reply:
why I am not supposed to export like this? is there is any main reason?

Reply:
@himanshujaidka The reason is to follow a process while developing a project with a group of individuals. The process is to follow __consistency__ and this is the process we have to follow as agreed upon by the project admin.

Reply:
I think nobody has declared the system and nobody approved, that how to write code and which files we have to use or not, and  I think that this way of exporting will not affect the project but it will lead to the efficiency of the project

Reply:
@purnima143 Can you please clarify this for him that following a proper structure throughout the api development necessary for the upcoming developers interested to contribute, and this how a team works on a project by following certain code style so that is is easier for debugging the code, and also easier for the frontend developers to understand it till we come up with a descriptive documentation. If a certain consistency in programming is not followed then it will be tougher for other developers who haven't written that code to debug it.